doc/data/messages/u/undefined-loop-variable/bad.py

```diff
@@ -0,0 +1,6 @@
+def find_even_number(lst):
+    for x in lst:
+        if x % 2 == 0:
+            break
+
+    return x  # [undefined-loop-variable]
```

Comment:
pylint raises two duplicate errors against line 6. Is this a bug?

```
> pylint doc/data/messages/u/undefined-loop-variable/bad.py
************* Module bad
doc/data/messages/u/undefined-loop-variable/bad.py:6:11: W0631: Using possibly undefined loop variable 'x' (undefined-loop-variable)
doc/data/messages/u/undefined-loop-variable/bad.py:6:11: W0631: Using possibly undefined loop variable 'x' (undefined-loop-variable)

------------------------------------------------------------------
Your code has been rated at 6.00/10 (previous run: 6.00/10, +0.00)
```

Reply:
```suggestion
def find_even_number(lst):
    for x in lst:
        if x % 2 == 0:
            break

    return x  # [undefined-loop-variable,undefined-loop-variable]
```

can make the test pass.

Reply:
This look like an issue, what do you think @jacobtylerwalls ?

Reply:
Let's hold this PR on the hope we can merge a fix relatively soon.

Reply:
@jacobtylerwalls fixed that on the main branch in #6925, could you merge main or rebase please @harupy ?

Reply:
rebased